docs/contributing/governance/veyfi.md

```diff
@@ -0,0 +1,57 @@
+# veYFI
+
+> This product is not deployed yet and is still under construction! You can follow development at https://github.com/yearn/veYFI
+
+veYFI incorporates [YIP-56: Buyback and Build](https://gov.yearn.finance/t/yip-56-buyback-and-build/8929) funds into YFI tokenomics: Users can lock YFI tokens and receive veYFI which allows them to boost vault rewards and vote on where bought-back YFI will be sent to.
+
+### Table-of-Content
+- [Specification](#specification)
+- [Benefits](#benefits)
+- [Risks](#risks)
+
+## Specification
+
+### veYFI
+
+- Receives bought-back YFI as rewards
+- Locking similar to the ve-style program of Curve.
+- YFI can be locked up to 4 years into veYFI, which is non-transferable.
```

Comment:
You say it can be locked up to 4 years, then right below it you say it hasn't been determined. I would consolidate the two points into 1.